src/invidious/comments.cr

```diff
@@ -552,7 +552,10 @@ def content_to_comment_html(content)
       if url = run["navigationEndpoint"]["urlEndpoint"]?.try &.["url"].as_s
         url = URI.parse(url)
 
-        if !url.host || {"m.youtube.com", "www.youtube.com", "youtu.be"}.includes? url.host
+        if !url.host
```

Comment:
If I'm not mistaken, `if !url.host` should be handled the same way as if the domain is `*.youtube.com`, in the case where google uses relative links

Reply:
Oops, thanks!